backend/src/govsg/services/govsg.service.ts

```diff
@@ -114,6 +115,18 @@ export function uploadCompleteOnPreview({
   }
 }
 
+const getLanguageCode = (language: string) => {
+  const languageInLowerCase = language.toLowerCase()
+  const whatsAppLanguages = Object.keys(WhatsAppLanguages)
+  const key = whatsAppLanguages.find(
+    (whatsAppLanguage) => whatsAppLanguage.toLowerCase() === languageInLowerCase
```

Comment:
If we look at the current situation, `toLowerCase()` is redundant here but I want to be resilient against careless refactors.